test/runtests.jl

```diff
@@ -3,7 +3,6 @@ using Test
 using Random
 using Statistics
 using SparseArrays
-using Arpack
 using LinearAlgebra
 #using Lint
```

Comment:
if you're making anymore changes maybe we can get rid of this line?

Reply:
Done...